src/components/NotificationDialog.vue

```diff
@@ -182,6 +182,7 @@ export default {
                 "SMSManager": "SmsManager (smsmanager.cz)",
                 "WeCom": "WeCom (企业微信群机器人)",
                 "ServerChan": "ServerChan (Server酱)",
+                "PushPlus": "PushPlus(推送加)",
```

Comment:
lets be more consistent with the others ^^
(I know that there are some outstanding formatting bugs for some)

```suggestion
                "PushPlus": "PushPlus (推送加)",
```